collector-bitmovin-player/src/test/java/com/bitmovin/analytics/bitmovin/player/utils/AdBreakMapperTest.kt

```diff
@@ -48,28 +49,28 @@ class AdBreakMapperTest {
     fun FromPlayerAdConfigurationWithAdConfigurationShouldSetValuesOfAdConfiguration(){
         val collectorAdBreak = adBreakMapper.FromPlayerAdConfiguration(adConfiguration)
 
-        assertThat(collectorAdBreak.replaceContentDuration).isEqualTo(TEN_SECONDS)
+        assertThat(collectorAdBreak.replaceContentDuration).isEqualTo(TEN_SECONDS_IN_MS)
     }
 
     @Test
     fun FromPlayerAdConfigurationWithAdConfigurationShouldNotSetValuesOfAdBreak(){
         val collectorAdBreak = adBreakMapper.FromPlayerAdConfiguration(adConfiguration)
 
-        assertThat(collectorAdBreak.scheduleTime).isEqualTo(TEN_SECONDS)
+        assertThat(collectorAdBreak.scheduleTime).isNull()
```

Comment:
it was a copy error
the scheduleTime prop is in the AdBreak but not in the AdConfiguration
because i feed an adConfiguration to the method there should not be a value